### DISCUSSION TITLE
Can I use Github CoPilot if I have a student benefits for github?

### DISCUSSION THREAD

**Discussion by BajajPratyush** [community/community#64589] [GitHub Education] [answered]
### Hi everyone,

I have been using github for a while and I recently saw about the benefits of github student account so I tried using a few of them but I cannot access Github copilot but I want to try it, can you tell how should I enable it?

**Comment by arafajs** [comment, marked as answer]
- Step 1: Visit the student developer Pack main page (education.github.com/pack)
- Step 2: Sign Up for the student developer pack
- Step 3: Select Get Student Benefit
- Step 4: Sign-in or SignUp for a GitHub account.
- Step 5: Add your school email. This process usually needs some authentication, because a code will be sent to your school email for verification.
- Step 6: Enter the name of your school
- Step 7: Answer the questions on how you plan to use GitHub.
- Step 8: On some occasions, more proof might be required. So you might need to upload either of school identity card or any other form of proof of study validity.
- Final step: Submit the application.

**Reply by BajajPratyush** [reply]
Thanks

**Reply by thisiskazem** [reply]
I've done the steps but is not activated, yet. Does it take time?

**Reply by aMIT-sHAHARE** [reply]
I am entering my college name but it's not working i.e continue button doesn't click ,how I can solve the problem

**Reply by dfeistauer** [reply]
> I've done the steps but is not activated, yet. Does it take time?

I have the same problem. How did you solve it?

**Comment by BenjitsuID** [comment]
I have a question about this,

I did the exact same thing and uploaded my student card  but it just keeps going back to the same screen and when i repeat the steps it goes the the exact same page this page btw (check the screenshot I added)
![studentcardfknSCREEN](https://github.com/community/community/assets/72789288/386ec848-ef04-4af3-85c1-515cad4719c8)

**Reply by MoAbbasid** [reply]
@BenjaminL-immalle 
Check that its JPG,  I had the same issue but found out I was trying to upload a PDF

**Reply by BenjitsuID** [reply]
Just tried again and it's still not working.
And yes its a JPG.

**Reply by Tucchhaa** [reply]
@BenjaminL-immalle I also had this problem. Your image has to be exactly in this format:
<img width="764" alt="image" src="https://github.com/community/community/assets/84278206/2a7ad920-9d7e-4bdf-9649-1dd9b1091db4">

**Reply by mdarl24** [reply]
When it complained my cellphone's picture size was over 1MB, I lowered the resolution using ImageMagick's `convert`.  I uploaded that, and it complained that the image was edited.  🤷 🤦  My university gives proof in PDF format.  It would be really nice if it took PDFs and if they raised the filesize to something more modern.

**Comment by mahui111** [comment]
I have the same issue,it doesn't allow me to upload  jpg proof, but ask me to take photo, Does anyone  know how to solve this?
